egs/iwslt22_ta/ST/RESULTS.md

```
@@ -0,0 +1,125 @@
+
+### IWSLT Tunisian training results  (Zipformer)
+
+#### 2023-06-01
+
+You can find a pretrained model, training logs, decoding logs, and decoding results at:
+
+
+
+|    Decoding method                 |     dev Bleu     |    test Bleu    | comment                                  |
+|------------------------------------|------------|------------|------------------------------------------|
+| modified beam search               | 14.7	    | 12.4       | --epoch 20, --avg 10, beam(10),pruned range 5 |
```

markdownlint-cli2 (0.17.2)

Hard tabs
Column: 44

(MD010, no-hard-tabs)